### ISSUE TITLE
Confirm that CARP and CBASS both use Rcpp::checkUserInterrupt()

### ISSUE THREAD

**Issue by alexpghayes** [DataSlingers/clustRviz#59]
For some long running operations, I was unable to stop the calculations once they started. Might want to check that `Rcpp::checkUserInterrupt()` gets called semi-frequently in the main looping operations as recommended in the Best Practices section of [R Packages](http://r-pkgs.had.co.nz/src.html).

**Comment by alexpghayes** [comment]
Yeah, same dataset. This could have been a combination of me being impatient and my computer being a potato.

**Comment by michaelweylandt** [comment]
FYI -- the current `develop` (post #65) tries to check for user interrupts every 0.2s. (Though for enormous problems it still won't do that since we won't interrupt in the middle of linear algebra steps) 

Unfortunately, the `develop` -> `master` automated build is busted for reasons I don't understand, so it's a bit tricky to try out right now.

**Comment by michaelweylandt** [comment]
The `develop` -> `master` build is now (finally...) working again, so you can test things out. 

With your data set: `CARP(tf, alg.type = "carp", X.scale = TRUE)` takes about 6 minutes on my machine, which is still a bit slow, but not insane.
